recipes/default.rb

```diff
@@ -20,8 +20,22 @@
 # prime the search to avoid 2 masters
 node.save unless Chef::Config[:solo]
 
-package 'drbd8-utils' do
-  action :install
+case node['platform']
```

Comment:
this should be platform_family which will include opensuse, opensuseleap, centos, scientific, oracle, and amazon